src/order/game_add.lean

```diff
@@ -67,6 +81,31 @@ begin
   exacts [iha _ ra (acc.intro b hb), ihb _ rb],
 end
 
-/-- The sum of two well-founded games is well-founded. -/
+/-- The `prod.game_add` relation is well-founded. -/
```

Comment:
Your docstring doesn't mention that rα and rβ need to be well-founded, while my original docstring captures that. In my opinion the statement of the lemma is clear enough so no need to repeat it in the docstring, so I simply comment its connection to combinatorial games in the docstring.
```suggestion
/-- The sum of two well-founded games is well-founded. -/
```

Reply:
Again, I would rather avoid referencing combinatorial games when it's not really relevant. It seems too specific of a reading for a relation on generic pairs.